internal/command/machine/leases.go

```diff
@@ -0,0 +1,201 @@
+package machine
+
+import (
+	"context"
+	"fmt"
+	"strings"
+	"time"
+
+	"github.com/spf13/cobra"
+	"github.com/superfly/flyctl/api"
+	"github.com/superfly/flyctl/client"
+	"github.com/superfly/flyctl/flaps"
+	"github.com/superfly/flyctl/internal/app"
+	"github.com/superfly/flyctl/internal/command"
+	"github.com/superfly/flyctl/internal/command/apps"
+	"github.com/superfly/flyctl/internal/config"
+	"github.com/superfly/flyctl/internal/flag"
+	"github.com/superfly/flyctl/internal/render"
+	"github.com/superfly/flyctl/iostreams"
+)
+
+func newLeases() *cobra.Command {
+	const (
+		short = "Manage machine leases"
+		long  = short + "\n"
+		usage = "leases <command>"
+	)
+
+	cmd := command.New(usage, short, long, nil)
+
+	cmd.Aliases = []string{"lease"}
+
+	cmd.Args = cobra.NoArgs
+
+	cmd.AddCommand(
+		newLeaseView(),
+		newLeaseClear(),
+	)
+
+	return cmd
+}
+
+func newLeaseView() *cobra.Command {
+	const (
+		short = "View machine leases"
+		long  = short + "\n"
+		usage = "view"
+	)
+
+	cmd := command.New(usage, short, long, runLeaseView,
```

Comment:
How can we make this a better experience for users who want to view/clear all leases at the same time?

Reply:
I'm honestly not sure what the best UX is for this.  I think this is a good start though, we can swing back to this later if we need to.